integrations/google-triggers-reference.md

```diff
@@ -331,10 +336,28 @@ We are pushing all available variables into the eCommerce tracking for more adva
       reservationRateName: "Best Price",
       checkInDate: "2022-08-29", // Check-in day of reservation
       checkOutDate: "2022-08-31", // Check-out day of reservation
-      tax: 1.3, // Tax of reservation
+      tax: 0.6, // Tax of reservation
       stayDuration: 2,  // Count of days in reservation
       adults: 2, // Count of adults in  reservation
       children: 0 // Count of children in reservation
+    },
+    {
+      item_id: "ec34da8b-0932-493e-a742-baf1af194789", // Product id
```

Comment:
In code, FE monorepo mews-js `packages/distributor/web-app/src/middlewares/integrationsMiddleware/purchaseEvent/transformation.ts`:

![Screenshot 2024-05-07 at 10 54 25](https://github.com/MewsSystems/gitbook-booking-engine/assets/6526378/d7e89a5a-d2d1-467b-96b9-8a12e227d400)

![Screenshot 2024-05-07 at 10 54 38](https://github.com/MewsSystems/gitbook-booking-engine/assets/6526378/cf4183c8-567d-4d3e-9b1d-e3e2868f9578)